cmd/convox/install.go

```diff
@@ -144,6 +144,29 @@ func init() {
 				Value: "10.0.3.0/24",
 				Usage: "Subnet 2 CIDR block",
 			},
+			cli.StringFlag{
+				Name:  "subnet-private0-cidr",
+				Value: "10.0.4.0/24",
+				Usage: "Private Subnet 0 CIDR block",
+			},
+			cli.StringFlag{
+				Name:  "subnet-private1-cidr",
+				Value: "10.0.5.0/24",
+				Usage: "Private Subnet 1 CIDR block",
+			},
+			cli.StringFlag{
+				Name:  "subnet-private2-cidr",
+				Value: "10.0.6.0/24",
+				Usage: "Private Subnet 2 CIDR block",
+			},
+			cli.BoolFlag{
+				Name:  "private",
+				Usage: "Create private network resources",
+			},
+			cli.BoolFlag{
+				Name:  "private-api",
+				Usage: "Put Rack API Load Balancer in private network. Implies --private",
```

Comment:
could we have a situation where only the rack wanted to be private but everything else wanted to be public?